tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Iast.cs

```diff
@@ -15,6 +15,12 @@ internal class Iast
             /// </summary>
             public const string Enabled = "DD_IAST_ENABLED";
 
+            /// <summary>
+            /// Configuration key for enabling or disabling the IAST Callsite Dataflow.
+            /// Default is value is true (enabled).
+            /// </summary>
+            public const string CallsiteEnabled = "DD_IAST_CALLSITE_ENABLED";
```

Comment:
Do we want a separate key for this? Eventually IAST will rely on call site being enabled,

Reply:
It was for finer grain testing, but I can remove it